### ISSUE TITLE
Such a dirty code？！

### ISSUE THREAD

**Issue by CrazyShout** [SizheWei/CoBEVFlow#6]
The code is confusing, the readme is confusing, and there are so many irrelevant files. Why doesn't the author update the code? This is my most troublesome code in 2024. I want to know if it can be reproduced by relying on this repo. Can the author please spend more time sorting it out?God bless you!

**Comment by JesseWong333** [comment]
I totally agree with you. The codebase is a mess and not reproducible.  Does the author carefully reproduce other works, e.g. Syncnet? Or only rush to publish a paper?

**Comment by liushiyangstd** [comment]
you are right !!!
